js/apps/admin-ui/public/locales/ru/authentication.json

```diff
@@ -18,9 +18,9 @@
     "none": "нет"
   },
   "flow": {
-    "browserFlow": "Сценарий браузера",
-    "registrationFlow": "Сценарий регистрации",
-    "directGrantFlow": "Сценарий Direct Grant Flow"
+    "browser": "Сценарий браузера",
+    "registration": "Сценарий регистрации",
+    "direct grant": "Сценарий Direct Grant Flow"
```

Comment:
we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"